src/resources/Pipelines/Pipelines.ts

```diff
@@ -16,15 +22,57 @@ export default class Pipelines extends Resource {
         this.associations = new MLAssociations(api);
     }
 
-    listBasicInfo() {
+    getBackendVersion() {
+        return this.api.get<PipelineBackendVersion>(
+            this.buildPath(`${Pipelines.searchUrlVersion2}/ml/version`, {organizationId: this.api.organizationId})
+        );
+    }
+
+    list(options?: ListPipelinesOptions) {
         return this.api.get<PipelineModel[]>(
-            this.buildPath(Pipelines.searchUrl, {organizationId: this.api.organizationId})
+            this.buildPath(Pipelines.searchUrlVersion1, {organizationId: this.api.organizationId, ...options})

```

Comment:
This will probably influence the pipeline selection in ML model testing. Did you verify if it returns the same results as previous call? And we might need a little update in `ModelTestingActions`

Reply:
I validated it returns the same results, and I tagged my commit as a breaking change, so it'll bump a major. That way we'll be good to make the change in the ModelTestingActions when we're ready.